app/scripts/contentscript.js

```diff
@@ -223,7 +223,6 @@ function blockedDomainCheck() {
  * Redirects the current page to a phishing information page
  */
 function redirectToPhishingWarning() {
-  console.log('MetaMask - routing to Phishing Warning component')
```

Comment:
This seemed useful :thinking: I like knowing why I'm being redirected - that it was MetaMask's doing, not a server.

Reply:
Restored to a `console.debug` in 31d1dbc